source/Halibut/IHalibutRuntime.cs

```diff
@@ -5,6 +5,12 @@
 
 namespace Halibut
 {
+    public enum HandleUnAuthorizedClientMode
```

Comment:
Consider `UnAuthorizedClientConnectAction`?

Also, the capitalization on the `A` is throwing me. I wonder if `Unauthorized` is better? A quick google says its a valid word, so no need to capitalize the `A`.

Reply:
Yep, agreed.